src/os/KeyValueStore.cc

```diff
@@ -1931,13 +1935,13 @@ int KeyValueStore::_generic_write(StripObjectMap::StripObjectHeaderRef header,
         value.append_zero(header->strip_size-value.length());
 
       header->bits[iter->no] = 1;
+      header->updated = true;
```

Comment:
header->updated has been updated in previous if statement, i think we need not update this field again